apps/client/models/email.model.ts

```diff
@@ -0,0 +1,7 @@
+export interface Email {
+  id: string
+  title: string
+  status: string
+  sentAt: string
+  content: string
+}
```

Comment:
minor thing, add empty space in the end of each file

Reply:
This interface could have field names based on[ server API Swagger](https://github.com/diconium/ethereal-pulse/blob/main/apps/server/src/config/swagger/swagger.yaml) fields for easy integration and consistency.

Reply:
On this PR I will left this as his and will improve it on the email dashboard issue